File: eform-client/src/app/plugins/modules/case-management-pn/models/calendar/users/calendar-users-request.model.ts
export class CalendarUsersRequestModel {
  public sort: string;
  nameFilter: string;
  pageIndex: number;
  pageSize: number;
  isSortDsc: boolean;
  offset: number;

  constructor() {
    this.sort = 'Id';
    this.isSortDsc = true;
    this.pageSize = 10;
    this.pageIndex = 0;
    this.offset = 0;
  }
}

File: eFormAPI/Plugins/CaseManagement.Pn/CaseManagement.Pn/Infrastructure/Models/Calendar/CalendarUsersRequestModel.cs
namespace CaseManagement.Pn.Infrastructure.Models.Calendar
{
    public class CalendarUsersRequestModel
    {
        public string Sort { get; set; } = "Id";
        public string NameFilter { get; set; }
        public int PageIndex { get; set; } = 0;
        public int PageSize { get; set; } = 10;
        public bool IsSortDsc { get; set; } = true;
        public int Offset { get; set; } = 0;
    }
}